bin/check-node-version.js

```diff
@@ -5,19 +5,29 @@ const { execSync } = require( 'child_process' );
 const fs = require( 'fs' );
 const path = require( 'path' );
 
+const getMajorVersion = ( version ) => {
+	const majorVersion = version.split( '.' )[ 0 ];
+
+	if ( majorVersion === undefined ) {
+		throw new Error( `Unable to parse version: ${version}` );
+	}
+
+	return parseInt( majorVersion );
```

Comment:
Let's include the radix, following best practises. I'll apply this myself to save the round trip:

```suggestion
	return parseInt( majorVersion, 10 );
```